matrix_io/malos/v1/maloseye.proto

```diff
@@ -29,6 +29,8 @@ option java_package = "one.matrixio.proto.malos.v1";
 message MalosEyeConfig {
   // Camera configuration. Camera id, width, height, and so on.
   CameraConfig camera_config = 1;
+  // Face configuration. models path, send slow, send text proto.
+  FaceConfig face_config = 2;
```

Comment:
Hey. I guess each field in the message definition has to have a unique number. The number 2 is used right below. Can you check it?

Reply:
2 must be unique into MalosEyeConfig message...  seems ok.

Reply:
Yeah. Did you open it? There's a int32 detection_server_timeout = 2; right below.

Reply:
Nevermind. It was the collapsing of github. My bad.